services/worker-manager/src/main.js

```diff
@@ -181,11 +181,17 @@ let load = loader({
     }),
   },
 
+  // This is used in testing to inject provider fakes
+  externalFakes: {
```

Comment:
What does "external" mean here?

Reply:
I was using it to mean "cloud service provider" I guess. Very open to suggestions for better names.

Reply:
fakeCloudApis?